### PR TITLE
[Merged by Bors] - Small test ux improvements

### PR DESCRIPTION
**Pull request by orndorffgrant**
Hi, first time contributor :)

I was playing with getting the test262 test runner working and found a couple small things that I thought would've made my life a bit easier. I've split them into separate commits. Just let me know if you disagree with any and we can drop that commit. And of course, let me know of any changes you'd like to see :)

The changes are:
- Adds details to the documentation on the `--suite` command to `boa_tester`. I was trying to pass a path starting with `./test262/` for a bit before I looked at the code and saw what it was doing.
- Changes the individual test output when verbosity is > 1. Because the tests are run in parallel, the "Result" line for a given test was frequently not immediately after the "Started" line. This made it hard to determine which test had failed. The new output includes the test name in the result line, and also changes the format of all the individual-test-output lines to begin with the test name.
- Adds a `--disable-parallelism` flag. Even with the adjustments to the test output, it was still a bit hard to follow. Running serially for small sub-suites produces output that can be more easily scanned IMO. I added this as a "disable" flag (as opposed to "enable parallelism") in order to maintain the current default of running in parallel.

### CODE REVIEW

**Review by jedel1043 — APPROVED** [review]
Good job! This should make it a lot easier to test for missing features and catching bugs :)

**Comment by orndorffgrant** [comment]
Thanks!

And I just noticed I missed a formatting error. Just pushed an update, hopefully it's green now

**Review by RageKnify — APPROVED** [review]

**Review by Razican — APPROVED** [review]
Looks good to me! In the future maybe we should have a log buffer in order to be able to parallelize and still get the correct output :)
